packages/echo/tsup.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'tsup';
+import { type SupportedFrameworkName } from './src';
+
+const frameworks: SupportedFrameworkName[] = ['h3', 'express', 'next', 'nuxt'];
+
+export default defineConfig({
+  entry: ['src/index.ts', ...frameworks.map((framework) => `src/${framework}.ts`)],
+  splitting: false,
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏ `ts-up` enables splitting of ESM by default, and disables CJS splitting. This flag explicitly disables splitting for ESM too.

Are we aware of any risks of splitting ESM files?

Without splitting:
```
> tsup

CLI Building entry: src/index.ts, src/h3.ts, src/express.ts, src/next.ts, src/nuxt.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v8.0.2
CLI Using tsup config: /Users/rifont/git/novu/packages/echo/tsup.config.ts
CLI Target: esnext
CLI Cleaning output folder
CJS Build start
ESM Build start
DTS Build start
CJS dist/express.cjs 7.80 KB
CJS dist/index.cjs   100.81 KB
CJS dist/h3.cjs      14.30 KB
CJS dist/next.cjs    8.75 KB
CJS dist/nuxt.cjs    14.30 KB
CJS ⚡️ Build success in 236ms
ESM dist/express.js 7.56 KB
ESM dist/next.js    8.51 KB
ESM dist/index.js   100.20 KB
ESM dist/h3.js      14.05 KB
ESM dist/nuxt.js    14.05 KB
ESM ⚡️ Build success in 237ms
DTS ⚡️ Build success in 2229ms
DTS dist/h3.d.cts      384.00 B
DTS dist/express.d.cts 265.00 B
DTS dist/next.d.cts    742.00 B
DTS dist/nuxt.d.cts    384.00 B
DTS dist/index.d.cts   185.01 KB
DTS dist/h3.d.ts       383.00 B
DTS dist/express.d.ts  264.00 B
DTS dist/next.d.ts     741.00 B
DTS dist/nuxt.d.ts     383.00 B
DTS dist/index.d.ts    185.01 KB
```

With splitting:
```
> tsup

CLI Building entry: src/index.ts, src/h3.ts, src/express.ts, src/next.ts, src/nuxt.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v8.0.2
CLI Using tsup config: /Users/rifont/git/novu/packages/echo/tsup.config.ts
CLI Target: esnext
CLI Cleaning output folder
CJS Build start
ESM Build start
DTS Build start
ESM dist/express.js        614.00 B
ESM dist/nuxt.js           521.00 B
ESM dist/chunk-6OADKKG5.js 10.05 KB
ESM dist/index.js          90.76 KB
ESM dist/h3.js             519.00 B
ESM dist/chunk-X7KXUFY7.js 6.65 KB
ESM dist/next.js           1.25 KB
ESM ⚡️ Build success in 199ms
CJS dist/next.cjs    8.75 KB
CJS dist/nuxt.cjs    14.30 KB
CJS dist/index.cjs   100.81 KB
CJS dist/h3.cjs      14.30 KB
CJS dist/express.cjs 7.80 KB
CJS ⚡️ Build success in 200ms
DTS ⚡️ Build success in 2160ms
DTS dist/h3.d.cts      384.00 B
DTS dist/express.d.cts 265.00 B
DTS dist/next.d.cts    742.00 B
DTS dist/nuxt.d.cts    384.00 B
DTS dist/index.d.cts   185.01 KB
DTS dist/h3.d.ts       383.00 B
DTS dist/express.d.ts  264.00 B
DTS dist/next.d.ts     741.00 B
DTS dist/nuxt.d.ts     383.00 B
DTS dist/index.d.ts    185.01 KB
```

Reply:
Did we opt-in for non-splitting as a safer choice? The difference in size is quite significant so maybe we should consider splitting by default until we hit any issues. Having said that, we need to establish some (manual for now) example apps in Next.js, Remix and Svelte Kit so that we can easily test if the app builds with our SDK.